src/services/notificationService.js

```diff
@@ -0,0 +1,117 @@
+import db from '../models';
+
+const { Notification } = db;
+
+/**
+ * A collection of methods that handles the database interactions
+ * for managing a Facility as an entity of the App.
+ *
+ * @class NotificationService
+ */
+class NotificationService {
+  /**
+   * Fetches a notification instance based on it's primary key.
+   * @static
+   * @param {integer} notificationId - Primary key of the notification to be fetched.
+   * @param {object} options - Additional query information
+   * @returns {Promise<array>} - An instance of notification table including it's relationships.
+   * @memberof NotificationService
+   */
+  static async findById(notificationId, options = {}) {
+    return await Notification.findByPk(notificationId, options);
+  }
+
+  /**
+   * Create a new notification instance.
+   * @static
+   * @param {Object} data - A notification object.
+   * @returns {Promise<Array>} A promise object with updated room properties.
+   * @memberof NotificationService
+   */
+  static async create(data) {
+    const { dataValues: notification } = await Notification.create(data);
+    return notification;
+  }
+
+  /**
+   * Fetches all notifications for a specific user - read or unread.
+   * @static
+   * @param {integer} userId - The user's Id.
+   * @param {integer} offset - An offset signifying where to start from - zero based.
+   * @param {integer} limit - The maximum number of notifications to be returned per call.
+   * @returns {Promise<array>} - An instance of notification table including it's relationships.
+   * @memberof NotificationService
+   */
+  static async fetchAll(userId, offset = 0, limit = 10) {
+    return await Notification.findAll({
```

Comment:
@King-Success .With regards to my comment on the `.eslintrc.json file`. Can you may use the suggestion below so that we don't have to modify the configuration.
```suggestion
    const notifications = await Notification.findAll({.....})
    return notifications
```

Reply:
Ok sure, will do